WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -25,4 +25,17 @@ def index
       end
     end
   end
+
+  def show
+    @person = Person.current.includes(:user, :ranksSingle, :ranksAverage, :competitions).find_by_wca_id(params[:id])
+    @previous_persons = Person.where(wca_id: params[:id]).where.not(subId: 1).order(:subId)
+    @ranks_single = @person.ranksSingle
+    @ranks_average = @person.ranksAverage
+    @events_competed_in = Event.where(id: (@ranks_single.map(&:eventId) + @ranks_average.map(&:eventId)).uniq)
+    @world_championship_podiums = @person.world_championship_podiums
+    @medals = @person.medals
+    @records = @person.records
+    @results = @person.results.includes(:competition, :event, :format, :round_type).order("Events.rank, Competitions.start_date DESC, RoundTypes.rank DESC")
+    params[:results_event] ||= @results.first.event.id
```

Comment:
`results_event` is not a very friendly thing to see in a url... could we just go with `event=` instead?